backends/platform/sdl/sdl.cpp

```diff
@@ -502,17 +503,42 @@ Common::String OSystem_SDL::getTextFromClipboard() {
 
 #if SDL_VERSION_ATLEAST(2, 0, 0)
 	char *text = SDL_GetClipboardText();
+#ifdef USE_TRANSLATION
```

Comment:
It is my understanding we use `ISO-8859-1` when `USE_TRANSLATION` is not defined. It's probably better to convert to that rather than using raw UTF-8 data which is bound to fail to render properly.
(the same applies to `setTextInClipboard`)

Reply:
Right. And that would be a very simple change to do (here, and in the macOS backend).